app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -283,6 +309,16 @@ class GroupConversationDetailsViewModel @Inject constructor(
         )
     }
 
+    //TODO: duplication here with ConversationViewModel, think about some progress mechanism ?
+    private fun executeWithProgress(request: suspend () -> Unit) {
```

Comment:
why not just extract that `executeWithProgress` somewhere where it will be available in both `ConversationViewModel` and here?

Reply:
the only option I see is to have some BaseViewModel having this withProgress flag, I added a TODO instead because BaseViewModel quickly becomes messy, so that maybe we could implement some better mechanism in the future and think about it during collective first instead